source/object-model/bindings.md

```diff
@@ -1,8 +1,12 @@
 A binding creates a link between two properties such that when one changes, the
 other one is updated to the new value automatically. Bindings can connect
-properties on the same object, or across two different objects. Unlike most other
-frameworks that include some sort of binding implementation, bindings in
-Ember.js can be used with any object, not just between views and models.
+properties on the same object, or across two different objects.
+
+Unlike most other frameworks that include some sort of binding implementation,
+bindings in Ember.js can be used with any object, not just between views and
```

Comment:
would it be more appropriate to say "between components and models"? or maybe we should just strike this sentence entirely - i'm not sure we need to say how great Ember is compared to other frameworks. thoughts?